frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -179,6 +163,7 @@ const TranslationPage = () => {
         contentArray.push({
           lineIndex,
           originalContent: content,
+          status: "Default",
```

Comment:
What's the default return value of `status` and if it has one, why hardcode it? If it doesn't have one, should it?

> need to include status in StoryTranslationContentDTO on a different pr. statuses are just Default for now
Do you mean on the frontend? How does it work if it's querying for the variable but the frontend isn't expecting it in the response?